From b9f4cd27708b786b669636c2e261ed06eea730ef Mon Sep 17 00:00:00 2001
From: Sicong Jia <sicong.jia@gumstix.com>
Date: Wed, 7 Feb 2018 16:22:17 -0800
Subject: [PATCH] Include uaccess.h to avoid compiler error

Implicit declaration of function 'copy_to_user' will cause an error
uaccess.h has to be included.

Signed-off-by: Sicong Jia <sicong.jia@gumstix.com>
---
 pwm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pwm.c b/pwm.c
index 3f10e4f..a7e062e 100644
--- a/pwm.c
+++ b/pwm.c
@@ -24,6 +24,7 @@
    Tobias Simon
 */
 
+#include <linux/uaccess.h>
 #include <linux/init.h>
 #include <linux/module.h>
 #include <linux/device.h>
-- 
2.7.4

